v4/mobile/react-native/FastLink/ios/Podfile

```diff
@@ -83,15 +40,6 @@ target 'FastLink' do
   end
 
   use_native_modules!
-
-  # Enables Flipper.
-  #
-  # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable these next few lines.
-  add_flipper_pods!
```

Comment:
Flipper causes a lot of issues, so I decided to de-integrate it and have the example app running.